src/main/java/com/flansmod/common/driveables/DriveableType.java

```diff
@@ -134,6 +136,9 @@
 	public int startSoundLength;
 	public String engineSound;
 	public int engineSoundLength;
+	
+	/**Track animation frames */
+	public int animFrames = 2;
```

Comment:
2 would correspond to 3 frames, with the idea that using the simple 3 frame method wouldn't require any changes to the vehicles config. However as this still requires extensive changes to the model itself, that's pretty pointless.